Examples/Framework/include/ActsExamples/EventData/Measurement.hpp

```diff
@@ -485,4 +529,25 @@ class VariableMeasurementProxy
   }
 };
 
+template <typename... Args>
+MeasurementContainer::VariableProxy
+MeasurementContainer::emplaceVariableMeasurement(std::uint8_t size,
+                                                 Args&&... args) {
+  VariableProxy meas = makeMeasurement(size);
+
+  meas.construct(std::forward<Args>(args)...);
+
+  return meas;
+}
+
+template <std::size_t Size, typename... Args>
+MeasurementContainer::FixedProxy<Size>
+MeasurementContainer::emplaceFixedMeasurement(Args&&... args) {
+  FixedProxy<Size> meas = makeMeasurement<Size>();
+
+  meas.construct(std::forward<Args>(args)...);
+
+  return meas;
+}
```

Comment:
I wonder if it might be easier to inline define these functions

Reply:
I don't think that's possible, because the actual types for `FixedProxy` and `VariableProxy` are defined only _after_ the class definition, so you run into instantiations of undefined templates. :(